lightwood/api/json_ai.py

```diff
@@ -635,6 +637,8 @@ def _add_implicit_values(json_ai: JsonAI) -> JsonAI:
     }
 
     for field_name, implicit_value in hidden_fields.items():
+        if getattr(json_ai, field_name):
```

Comment:
While this is not needed, I do think it's good in case a user wants to specify "hidden" module args beforehand.